graph.go

```diff
@@ -372,7 +372,7 @@ func treePath(path string) string {
 	if path == "" || path == "." {
 		return string(".")
 	}
-	return string(fmt.Sprintf("./%s", path))
+	return string(fmt.Sprintf(".%s%s", filepath.Separator, path))
```

Comment:
For consistency with above (line 148) and clarity, can you rewrite this like this:

```Go
return "." + string(filepath.Separator) + path
```